package.json

```diff
@@ -36,7 +35,7 @@
     "stats-webpack-plugin": "^0.2.1",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.9.11",
+    "webpack": "^2.5.1",
```

Comment:
when testing it i got:
```
error extract-text-webpack-plugin@2.1.0: The engine "node" is incompatible with this module. Expected version ">=4.3.0 < 5.0.0 || >= 5.10".
error Found incompatible module
```

any idea what are the new minimum requirements? I assume this is going to effect our build infra too.
/cc @tbrisker